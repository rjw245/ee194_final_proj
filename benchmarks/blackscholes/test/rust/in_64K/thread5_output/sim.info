{'cmdline': ['/h/hzhou01/comp140/sniper/run-sniper',
             '-d',
             './thread5_output',
             '-n',
             '8',
             '-c',
             'gainestown',
             '-c',
             'nehalem.cfg',
             '--',
             '../../../rust/bs',
             '5',
             '../../../inputs/in_64K.txt'],
 'git_revision': '',
 'host': 'dell24',
 'pin_version': 71313,
 'rusage': (54103.455024,
            488.291768,
            170684,
            0,
            0,
            0,
            441700,
            0,
            0,
            0,
            12328,
            0,
            0,
            0,
            36903184,
            132171),
 'snipercmd': ['bash',
               '-c',
               '/h/hzhou01/comp140/sniper/pin_kit/intel64/bin/pinbin -mt -injection child -xyzzy -enable_vsm 0 -t /h/hzhou01/comp140/sniper/lib/pin_sim -c /h/hzhou01/comp140/sniper/config/base.cfg --general/total_cores=8 --general/output_dir=/h/hzhou01/ee194_final_proj/benchmarks/blackscholes/test/rust/in_64K/thread5_output --config=/h/hzhou01/comp140/sniper/config/nehalem.cfg --config=/h/hzhou01/comp140/sniper/config/gainestown.cfg --config=/h/hzhou01/comp140/sniper/config/nehalem.cfg -g --traceinput/stop_with_first_app=true -g --traceinput/restart_apps=false -- ../../../rust/bs 5 ../../../inputs/in_64K.txt'],
 't_elapsed': 24675.117785930634,
 't_start': 1462232497.533545,
 'tracecmds': [],
 'user': 'hzhou01',
 'vmem': 3862093824L}
