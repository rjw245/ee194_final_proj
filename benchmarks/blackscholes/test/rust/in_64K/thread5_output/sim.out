                                     | Core 0      | Core 1      | Core 2      | Core 3      | Core 4      | Core 5      | Core 6      | Core 7     
  Instructions                       |  4219242220 |  2661735752 |  2661747909 |  2661735925 |  2661732384 |  2661730752 |           0 |           0
  Cycles                             |  4755707068 |  4755707068 |  4755707068 |  4755707068 |  4755707068 |  4755707068 |  4755707068 |  4755707068
  IPC                                |        0.89 |        0.56 |        0.56 |        0.56 |        0.56 |        0.56 |        0.00 |        0.00
  Time (ns)                          |  1787859800 |  1787859800 |  1787859800 |  1787859800 |  1787859800 |  1787859800 |  1787859800 |  1787859800
  Idle time (ns)                     |   853602448 |   934238459 |   938528723 |   949065097 |   948805678 |   949676509 |  1787859800 |  1787859800
  Idle time (%)                      |       47.7% |       52.3% |       52.5% |       53.1% |       53.1% |       53.1% |      100.0% |      100.0%
Branch predictor stats               |             |             |             |             |             |             |             |            
  num correct                        |   148311980 |    93671572 |    94040843 |    94146519 |    94144859 |    94156568 |           0 |           0
  num incorrect                      |     5831837 |     2899028 |     2528212 |     2422286 |     2423986 |     2412643 |           0 |           0
  misprediction rate                 |       3.78% |       3.00% |       2.62% |       2.51% |       2.51% |       2.50% |       0.00% |       0.00%
  mpki                               |        1.38 |        1.09 |        0.95 |        0.91 |        0.91 |        0.91 |        0.00 |        0.00
TLB Summary                          |             |             |             |             |             |             |             |            
  I-TLB                              |             |             |             |             |             |             |             |            
    num accesses                     |   903861063 |   436945497 |   436944500 |   436943969 |   436943996 |   436943738 |           0 |           0
    num misses                       |         195 |          75 |          71 |          71 |          71 |          76 |           0 |           0
    miss rate                        |       0.00% |       0.00% |       0.00% |       0.00% |       0.00% |       0.00% |       0.00% |       0.00%
    mpki                             |        0.00 |        0.00 |        0.00 |        0.00 |        0.00 |        0.00 |        0.00 |        0.00
  D-TLB                              |             |             |             |             |             |             |             |            
    num accesses                     |  2813827183 |  1530320554 |  1530333841 |  1530325124 |  1530322257 |  1530320840 |           0 |           0
    num misses                       |      218702 |        7197 |        8006 |        8004 |        8006 |        7150 |           0 |           0
    miss rate                        |       0.01% |       0.00% |       0.00% |       0.00% |       0.00% |       0.00% |       0.00% |       0.00%
    mpki                             |        0.05 |        0.00 |        0.00 |        0.00 |        0.00 |        0.00 |        0.00 |        0.00
  L2 TLB                             |             |             |             |             |             |             |             |            
    num accesses                     |      218897 |        7272 |        8077 |        8075 |        8077 |        7226 |           0 |           0
    num misses                       |         824 |         291 |         240 |         237 |         244 |         258 |           0 |           0
    miss rate                        |       0.38% |       4.00% |       2.97% |       2.93% |       3.02% |       3.57% |       0.00% |       0.00%
    mpki                             |        0.00 |        0.00 |        0.00 |        0.00 |        0.00 |        0.00 |        0.00 |        0.00
Cache Summary                        |             |             |             |             |             |             |             |            
  Cache L1-I                         |             |             |             |             |             |             |             |            
    num cache accesses               |   903861063 |   436945497 |   436944500 |   436943969 |   436943996 |   436943738 |           0 |           0
    num cache misses                 |    22987473 |       15156 |       15157 |       14764 |       15415 |       14761 |           0 |           0
    miss rate                        |       2.54% |       0.00% |       0.00% |       0.00% |       0.00% |       0.00% |        inf% |        inf%
    mpki                             |        5.45 |        0.01 |        0.01 |        0.01 |        0.01 |        0.01 |         inf |         inf
  Cache L1-D                         |             |             |             |             |             |             |             |            
    num cache accesses               |  2813827858 |  1530320796 |  1530334102 |  1530325387 |  1530322516 |  1530321093 |           0 |           0
    num cache misses                 |      103172 |      575007 |      576454 |      573479 |      574855 |      573730 |           0 |           0
    miss rate                        |       0.00% |       0.04% |       0.04% |       0.04% |       0.04% |       0.04% |        inf% |        inf%
    mpki                             |        0.02 |        0.22 |        0.22 |        0.22 |        0.22 |        0.22 |         inf |         inf
  Cache L2                           |             |             |             |             |             |             |             |            
    num cache accesses               |    23090816 |      590168 |      591625 |      588260 |      590315 |      588512 |           0 |           0
    num cache misses                 |       39370 |      536900 |      538675 |      535784 |      537836 |      536748 |           0 |           0
    miss rate                        |       0.17% |      90.97% |      91.05% |      91.08% |      91.11% |      91.20% |        inf% |        inf%
    mpki                             |        0.01 |        0.20 |        0.20 |        0.20 |        0.20 |        0.20 |         inf |         inf
  Cache L3                           |             |             |             |             |             |             |             |            
    num cache accesses               |       39551 |      536905 |      538689 |      535801 |      537881 |      536769 |           0 |           0
    num cache misses                 |       33863 |         658 |         400 |         617 |        6193 |        5940 |           0 |           0
    miss rate                        |      85.62% |       0.12% |       0.07% |       0.12% |       1.15% |       1.11% |        inf% |        inf%
    mpki                             |        0.01 |        0.00 |        0.00 |        0.00 |        0.00 |        0.00 |         inf |         inf
DRAM summary                         |             |             |             |             |             |             |             |            
  num dram accesses                  |       18064 |           0 |           0 |           0 |       18068 |           0 |           0 |           0
  average dram access latency (ns)   |       57.99 |         inf |         inf |         inf |       58.12 |         inf |         inf |         inf
  average dram queueing delay        |        4.56 |        0.00 |        0.00 |        0.00 |        4.69 |        0.00 |        0.00 |        0.00
  average dram bandwidth utilization |       0.00% |       0.00% |       0.00% |       0.00% |       0.00% |       0.00% |       0.00% |       0.00%
Coherency Traffic                    |             |             |             |             |             |             |             |            
  num loads from dram                |        3837 |         148 |          57 |          71 |          52 |          58 |           0 |           0
  num loads from dram cache          |           0 |           0 |           0 |           0 |           0 |           0 |           0 |           0
  num loads from remote cache        |          19 |          12 |          36 |          75 |        5161 |        5170 |           0 |           0
